### PR TITLE
Move GOPATH check to a phony target

### PR DESCRIPTION
**Pull request by mrunalp**
Signed-off-by: Mrunal Patel <mrunalp@gmail.com>

### CODE REVIEW

**Comment by mrunalp** [comment]
@runcom @jawnsy @nalind PTAL

**Comment by runcom** [comment]
:+1:

File: Makefile [hunk 1]
```diff
@@ -30,10 +30,12 @@ help:
 	@echo " * 'lint' - Execute the source code linter"
 	@echo " * 'gofmt' - Verify the source code gofmt"
 
-lint:
+ensure-gopath:
```
Comment (jawnsy) [inline Makefile]: Suggest the name 'check-gopath' instead.

Also, we could put the .PHONY tags above each target, to make it easier to keep 'em in sync?

**Comment by mrunalp** [comment]
@jawnsy renamed. Will fixup existing .PHONY in a follow on.

**Comment by nalind** [comment]
LGTM

**Pull request merged**